src/base/bittorrent/tracker.cpp

```diff
@@ -272,6 +272,12 @@ void Tracker::processAnnounceRequest()
     announceReq.socketAddress = m_env.clientAddress;
     announceReq.claimedAddress = queryParams.value(ANNOUNCE_REQUEST_IP);
 
+    // Enforce using IPv4 if address is indeed IPv4 or if it is an IPv4-mapped IPv6 address
+    bool ok = false;
+    const qint32 decimalIPv4 = announceReq.socketAddress.toIPv4Address(&ok);
+    if(ok)
```

Comment:
And you still violate our coding style... there is a space between if and the left parentheses.

Reply:
@Chocobo1 
Fixed. I ran uncrustify with the current config on the file and also found a bunch of other warnings. Should I include the fixes in this PR?

Reply:
As a related question, in case of conflict between the coding guidelines and the uncrustify style, which is most authoritative? I assume (and hope) the uncrustify config, because I want to "fire and forget", not have to check every addition manually against the guidelines.

Reply:
>in case of conflict between the coding guidelines and the uncrustify style, which is most authoritative? 

Coding guidelines.

Reply:
>in case of conflict between the coding guidelines and the uncrustify style, which is most authoritative?

Of course, Coding Guidelines.
uncrustify config is written by some "lazy" contributors to simplify their life but it still not completely represents qBittorrent Coding Guidelines.
Really our coding style isn't so hard to understand. Basically, it causes problems only for those who do not adhere to any particular style at all, but write stylistically inconsistent code.